toptal/picasso
# fix(OverlayBadge): [SPC-1728] fix number alignment on small variant

[SPC-1728]

### Description

Fix the number alignment for the `OverlayBadge` component on the `small` variant.

### How to test

- Check the `small` variant to the `OverlayBadge` component
- Change the page zoom level to 125%
- The number alignment should keep on center

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="177" alt="Screen Shot 2022-03-24 at 16 23 24" src="https://user-images.githubusercontent.com/4870595/159994552-2786e7a9-3567-4004-864a-33e4a599b0be.png"> | <img width="165" alt="Screen Shot 2022-03-24 at 16 23 47" src="https://user-images.githubusercontent.com/4870595/159994595-ab82986d-5b4f-4e69-9c04-7f2f9d62a46b.png"> |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[SPC-1728]: https://toptal-core.atlassian.net/browse/SPC-1728?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
